docs/install_guides/pypi-openvino-dev.md

```diff
@@ -170,11 +170,11 @@ alias pip='noglob pip'
 
 On Windows*, some libraries are necessary to run OpenVINO. To resolve this issue, install the [C++ redistributable (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe). You can also view a full download list on the [official support page](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).
 
-### ImportError: libpython3.7m.so.1.0: cannot open shared object file: No such file or directory
+### ImportError: libpython3.10m.so.1.0: cannot open shared object file: No such file or directory
```

Comment:
should not we remove this section at all?
At least, `3.10m` is not valid configuration

Reply:
Good point, I think that this section is not needed as the issue is not OpenVINO related. I removed it from our PyPI pages.